scripts/run_gcosmos.sh

```diff
@@ -5,13 +5,77 @@ G_HTTP_ADDR=${G_HTTP_ADDR:-"26657"}
 G_GRPC_ADDR=${G_GRPC_ADDR:-"9092"}
 
 # cleanup previous run data as gordian can only start from height 0 currently
```

Comment:
This comment is no longer true on main; we now correctly handle starting from any height.

I'm fine either way of changing it in this PR or separately.

Reply:
handled